kgraphql-ktor/src/test/kotlin/com/apurebase/kgraphql/KtorConfigurationTest.kt

```diff
@@ -1,38 +0,0 @@
-package com.apurebase.kgraphql
-
-import com.apurebase.kgraphql.schema.execution.Executor
-import io.ktor.server.application.install
-import io.ktor.server.testing.testApplication
-import org.amshove.kluent.shouldBeEqualTo
-import org.junit.jupiter.api.Test
-
-class KtorConfigurationTest : KtorTest() {
-
-    @Test
-    fun `default configuration should`() {
-        var checked = false
-        testApplication {
-            application {
-                val config = install(GraphQL) {}
-                checked = true
-                config.schema.configuration.executor shouldBeEqualTo Executor.Parallel
```

Comment:
Not sure what exactly this was supposed to test but the return type of `install(GraphQL)` has changed and the schema is no longer available. Hopefully this is not a problem 🤔